scss/tokens/_raw.scss

```diff
@@ -17,3 +17,34 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .80 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-shadow
+// Box-shadow
+$ouds-elevation-x-0:           0 !default;
```

Comment:
Waiting for more information, but there could be also a `$ouds-elevation-x-200`.